### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/dexidp-website/actions/runs/4786382385#summary-12974645977

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/dexidp-website/actions/runs/4786382488#summary-12974645969

### CODE REVIEW

File: content/docs/connectors/github.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ connectors:
     # authenticate with dex.
     #
     # If neither 'org' nor 'orgs' are specified in the config and 'loadAllGroups' setting set to true then user
-    # authenticate with ALL user's Github groups. Typical use case for this setup:
+    # authenticate with ALL user's GitHub groups. Typical use case for this setup:
```
Comment (jsoref) [inline content/docs/connectors/github.md:57]: brand

File: content/docs/connectors/oauth.md [hunk 2]
```diff
@@ -48,8 +48,8 @@ connectors:
     # Default: id
     # userIDKey:
 
-    # Auth roviders return non-standard user identity profile
-    # Use claimMapping to map those user infomations to standard claims:
+    # Auth providers return non-standard user identity profile
+    # Use claimMapping to map those user informations to standard claims:
```
Comment (jsoref) [inline content/docs/connectors/oauth.md:52]: I'm not a huge fan of `informations` (I prefer using `information` as a mass noun), but using that would require changing `those` to `that` which is more than a simple spelling fix

**Review by jsoref — COMMENTED** [review]
Most corrections automatically suggested by Google Sheets. All fault mine.

**Review by sagikazarmark — APPROVED** [review]
Oh, wow. I had no idea we had so many typos.

Thanks!

**Comment by jsoref** [comment]
fwiw, this is quite light. Some larger things can have 1000+.

**Pull request merged**